src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,32 @@
+import { Slider } from '@sharcoux/slider';
+import type { SliderProps } from '@sharcoux/slider/dist/Slider';
+import { memo, useMemo } from 'react';
+import { useComponentStyles } from '../../hooks';
+
+export type Props = SliderProps & {};
```

Comment:
colors from `SliderProps` are not normalized. They should be abstracted and passed down into useComponentStyle's second argument